app/services/migrate_user_service.rb

```diff
@@ -0,0 +1,101 @@
+class MigrateUserService
+  def initialize(new_id:, old_id:)
+    @new_id = new_id
+    @old_id = old_id
+    @lines = []
+  end
+
+  def schema
+    {
+      visits: :user_id,
+      ahoy_events: :user_id,
+      ahoy_messages: :user_id,
+      groups: :creator_id,
+      memberships: [:user_id, :inviter_id],
```

Comment:
How will you handle the case of the @old_id and @new_id already belonging to the same group? Seems like that would violate the new [:user_id, :group_id] constraint on Membership